tests/h/views/api/config_test.py

```diff
@@ -10,36 +10,52 @@
 @pytest.mark.usefixtures("cors")
 class TestAddApiView(object):
     def test_it_sets_accept_setting(self, pyramid_config, view):
-        api_config.add_api_view(pyramid_config, view, route_name="thing.read")
+        api_config.add_api_view(
+            pyramid_config, view, versions=["v1"], route_name="thing.read"
+        )
         (_, kwargs) = pyramid_config.add_view.call_args_list[0]
         assert kwargs["accept"] == "application/json"
```

Comment:
Shouldn't adding the `versions=["v1"]` specifier here cause the `accept` setting to change?

Reply:
Ah wait, this is because `config.add_view` gets called twice, once with the `accept` setting and once without. We do have tests below that check that the _second_ `config.add_view` call sets the version-specific value for `accept`. I missed this because there are a bunch of tests unrelated to Accept headers in-between.

It also seems to me that we're missing a test here for what happens if multiple versions are specified?

Reply:
> It also seems to me that we're missing a test here for what happens if multiple versions are specified?

The app only knows about one version right at the moment, so it would throw. Effectively, at this point, it would be same test as testing what happens if you specify an undefined version (which already exists). Put another way, there's no way to specify a list of versions at present with more than one entry (without throwing) because the app only knows about one version.

This test will certainly get added in the next-step PR when I introduce "v2".